samples/place-autocomplete-data-session/index.ts

```diff
@@ -12,25 +12,23 @@ let token;
 
 // Add an initial request body.
 let request = {
-    input: "",
+    input: '',
     locationRestriction: { west: -122.44, north: 37.8, east: -122.39, south: 37.78 },
     origin: { lat: 37.7893, lng: -122.4039 },
-    includedPrimaryTypes: ["restaurant"],
-    language: "en-US",
-    region: "us",
+    includedPrimaryTypes: ['restaurant'],
+    language: 'en-US',
+    region: 'us',
 };
 
 async function init() {
-    token = new google.maps.places.AutocompleteSessionToken();
-
     title = document.getElementById('title');
     results = document.getElementById('results');
-    input = document.querySelector("input");
-    input.addEventListener("input", makeAcRequest);
-    request = refreshToken(request) as any;
+    input = document.querySelector('input');
+    input.addEventListener('input', makeAutocompleteRequest);
+    await refreshToken(request);
 }
 
-async function makeAcRequest(input) {
+async function makeAutocompleteRequest(input) {
```

Comment:
is there a race condition in this function?

e.g. if network conditions cause an earlier request to return later?